docs/static/breaking-changes.asciidoc

```diff
@@ -24,6 +25,21 @@ See also <<releasenotes>>.
 //NOTE: The notable-breaking-changes tagged regions are re-used in the
 //Installation and Upgrade Guide
 
+[[breaking-8.0]]
```

Comment:
@karenzone I should cc you before merging it :fearful:
Could you review the doc? This PR is a breaking change of removing `--field-reference-parser`